cinema/models.py

```diff
@@ -63,9 +63,7 @@ def __str__(self):
 
 class Order(models.Model):
     created_at = models.DateTimeField(auto_now_add=True)
-    user = models.ForeignKey(
-        settings.AUTH_USER_MODEL, on_delete=models.CASCADE
-    )
+    user = models.ForeignKey(settings.AUTH_USER_MODEL, on_delete=models.CASCADE)
```

Comment:
Ensure that the `settings.AUTH_USER_MODEL` is correctly set in your Django settings. This is crucial for the ForeignKey relationship to the user model.